src/include/packet_types.h

```diff
@@ -153,6 +153,14 @@ typedef struct altitude_data_block {
     uint8_t bytes[16];
 } TIGHTLY_PACKED AltitudeDataBlock;
 
-void altitude_data_block_init(AltitudeDataBlock *b, const int8_t measurment_type, const uint8_t pressure,
-                              const uint8_t temperature, const uint8_t altitude);
-#endif // _PACKET_TYPES_H
+void altitude_data_block_init(AltitudeDataBlock *b, const uint32_t measurement_time, const int32_t pressure,
+                              const uint32_t temperature, const uint32_t altitude);
+
+typedef struct angular_velocity_block {
+    /**The angular velocity block accessef as a bytes array*/
+    uint8_t bytes[12];
+} TIGHTLY_PACKED AngularVelocityBlock;
+
+void angular_velocity_block(AngularVelocityBlock *b, const uint32_t measurement_time, const int8_t full_scale_range,
```

Comment:
This needs to have the same name as your definition, `angular_velocity_block_init`.